test/extensions/filters/udp/dns_filter/dns_filter_test.cc

```diff
@@ -21,66 +21,125 @@ namespace UdpFilters {
 namespace DnsFilter {
 namespace {
 
+Api::IoCallUint64Result makeNoError(uint64_t rc) {
+  auto no_error = Api::ioCallUint64ResultNoError();
+  no_error.rc_ = rc;
+  return no_error;
+}
+
 class DnsFilterTest : public testing::Test {
 public:
   DnsFilterTest()
       : listener_address_(Network::Utility::parseInternetAddressAndPort("127.0.2.1:5353")) {
+    // TODO: Consume the log setting from the command line
+    // Logger::Registry::setLogLevel(TestEnvironment::getOptions().logLevel());
+    Logger::Registry::setLogLevel(spdlog::level::trace);
```

Comment:
Delete. You can set test level on the command line now via `--test_arg="-l trace"`

Reply:
Yep.  I got that working elsewhere.  I will update it here.